src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -387,7 +388,7 @@ export function getPrismaClient(config: GetPrismaClientOptions): any {
         '.env',
       )
       if (fs.existsSync(dotEnvPath)) {
-        return parseDotEnv(fs.readFileSync(dotEnvPath, 'utf-8'))
+        return dotenvExpand(parseDotEnv(fs.readFileSync(dotEnvPath, 'utf-8')))
```

Comment:
Needed ? `dotenv-expand` expects config as a parameter, not `parse`'s result https://github.com/motdotla/dotenv-expand/blob/master/lib/main.js#L3

Reply:
Indeed I will fix that before merging